src/librustdoc/html/render/print_item.rs

```diff
@@ -1618,6 +1618,9 @@ fn item_variants(
         if let clean::VariantKind::Tuple(ref s) = variant_data.kind {
             write!(w, "({})", print_tuple_struct_fields(cx, s));
         }
+        if variant.attrs.other_attrs.iter().any(|a| a.has_name(kw::Default)) {
+            w.write_str(" <span class=\"stab default\">Default</span>");
```

Comment:
Reusing the CSS class `stab` (for stability “chips”) doesn't seem ideal to me, what about renaming `stab` to `chip` or splitting the CSS class into two?

Reply:
done